benchmark/benchmark.php

```diff
@@ -118,7 +118,8 @@ function runValgrindPhpCgiCommand(
         '-T' . ($warmup ? $warmup . ',' : '') . $repeat,
         '-d max_execution_time=0',
         '-d opcache.enable=1',
-        '-d opcache.jit_buffer_size=' . ($jit ? '128M' : '0'),
+        '-d opcache.jit=' . ($jit ? 'tracing' : 'disable'),
```

Comment:
Shy not just enable/disable?

Reply:
IMO it's better to explicitly specify the JIT mode.